python/ray/tests/test_global_state.py

```diff
@@ -378,6 +378,54 @@ def backlog_size_set():
     global_state_accessor.disconnect()
 
 
+def test_default_load_reports(shutdown_only):
+    """Despite the fact that default actors release their cpu after being
```

Comment:
Could we also add a fake autoscaler test with one of these actors? Can copy one from the existing fake autoscaler suite.